lib/sles4sap_publiccloud.pm

```diff
@@ -920,9 +934,9 @@ sub create_playbook_section_list {
     if ($args{fencing} eq 'native' and is_azure) {
         # Prepares Azure native fencing related arguments for 'sap-hana-cluster.yaml' playbook
         my $azure_native_fencing_args = azure_fencing_agents_playbook_args(
-            fence_type => get_var('AZURE_FENCE_AGENT_CONFIGURATION', 'msi'),
```

Comment:
half of the arguments was from `%args` and half of them was from `get_var`. I decided to move all of them to `%args`.